jobs-tests/src/test/java/org/zowe/jobs/tests/JobsGetIntegrationTest.java

```diff
@@ -56,6 +56,18 @@ public void testGetJobs() {
             assertThat(job.getOwner(), IsEqualIgnoringCase.equalToIgnoringCase(USER));
         }
     }
+    
+    @Test
+    public void testGetJobsNoOwnerShouldDefaultToTestUser() {
+        List<Job> actual = getJobs("*", null).then().statusCode(HttpStatus.SC_OK).extract().body().jsonPath()
+            .getList("items", Job.class);
+
+        // We have results, they are of type job and all have owner = user
+        assertTrue(actual.size() > 0);
+        for (Job job : actual) {
+            assertThat(job.getOwner(), IsEqualIgnoringCase.equalToIgnoringCase(USER));
+        }
+    }
```

Comment:
Is this pretty much a duplicate of testGetJobsWithCurrentUserAsOwnerAndSpecificPrefix()?